packages/flutter/lib/src/material/icon_button.dart

```diff
@@ -34,10 +37,23 @@ class IconButton extends StatelessComponent {
         )
       )
     );
+    if (tooltip != null) {
+      result = new Tooltip(
+        message: tooltip,
+        child: result
+      );
+    }
+    return result;
   }
 
   void debugFillDescription(List<String> description) {
     super.debugFillDescription(description);
     description.add('$icon');
+    if (onPressed == null)
+      description.add('disabled');
+    if (color != null)
+      description.add('$color');
```

Comment:
When you stringify a color it already has "Color" in the string so I often skip the 'color:' for widget descriptions (since they all end up on one line brevity is important).